article_maker.cc

```diff
@@ -149,20 +149,19 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word,
             "</script>";
   result+= R"(<script type="text/javascript" src="qrc:///scripts/gd-builtin.js"></script>)";
 
-#ifdef Q_OS_WIN32
```

Comment:
`#ifdef Q_OS_WIN32`
this line is needed .   in some cases ,users may migrate the configuration file from windows to linux.

Reply:
This line now depends  on option `darkReaderMode` not `darkMode` anymore

Reply:
Remove this line, so that mac and linux users may also enable darkreader